templates/Parameters.py.template

```diff
@@ -0,0 +1,134 @@
+# *********************************************************
+#
+# File autogenerated for the ${pkgname} package
+# by the rosparam_handler package.
+# Please do not edit.
+#
+# *********************************************************
+import rospy
+
+param_description = $paramDescription
```

Comment:
I am not up to date with python anymore: Could we get a problem here, if a rosnode includes multiple Param files, because param_description is a global variable?

Reply:
Only if someone repeatedly did `from <paramfile> import *` which is generally discuraged in python.

I borrowed this style from [dynamic_reconfigure](https://github.com/ros/dynamic_reconfigure/blob/master/templates/ConfigType.py.template) - they do that too.

Reply:
Well I guess that's ok then.